frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.styled.tsx

```diff
@@ -25,4 +25,7 @@ export const Label = styled.span`
 export const BoldExternalLink = styled(ExternalLink)`
   color: var(--mb-color-brand);
   font-weight: bold;
+  display: flex;
+  align-items: center;
+  gap: calc(6 / 16 * 1rem);
```

Comment:
also curious about this - this is gonna be 0.375rem - I would push back and ask design if 0.5rem is ok

Reply:
This is from the old design when we still had the link icon for embedding SDK. The distance measured in Figma from the icon to the edge of the text was `6.25px`.

![image](https://github.com/user-attachments/assets/54018e94-1250-45f6-988e-88539addb07c)


I must say that changing this to `8px` or `0.5rem` 
![image](https://github.com/user-attachments/assets/f42a35cf-8b51-4edb-a972-ccf7f8698dbc) doesn't make a huge difference unless you're really picking.

@AlessioLaiso I see in the latest design this icon has been removed because the whole link in the SDK card is removed. Should I still have it for interactive embedding or leave it as is (no icon).